src/plugins/dam/components/uploader/FileList.jsx

```diff
@@ -65,6 +55,34 @@ class FileList extends React.Component {
     }
   }
 
+  ensureActiveItemVisible() {
+    if (!this.activeFileItem) {
+      return
+    }
+
+    const currentClassName = this.damFileList.current.className;
+
+    computeScrollIntoView(this.activeFileItem, { scrollMode: 'if-needed' }).forEach((action) => {
+      if (action.el.className !== currentClassName) {
+        return
+      }
+      
+      const el = action.el;
+      const top = action.top;
+
+      if (el.scroll && ('scrollBehavior' in document.body.style)) {
+        el.scroll({
+          top,
+          behavior: 'smooth',
+        });
+      } else {
+        el.scrollTop = top;
+      }
+    });
+
+    throttle(() => throttledComputeScrollIntoView(), 5000);
```

Comment:
@digitalartcrew 
- you dont have to "double function" this. `() => someFunction()` is the same thing as `someFunction`. the end result is the same
- doesnt look like `throttledComputeScrollIntoView` exists